code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -152,6 +148,16 @@
 	if(istype(target, /obj/vehicle/sealed/mecha))
 		var/obj/vehicle/sealed/mecha/M = target
 		M.take_damage(50, BRUTE, MELEE, 1)
+	
+/mob/living/simple_animal/hostile/space_dragon/RangedAttack(atom/target, modifiers)
+	if(LAZYACCESS(modifiers, RIGHT_CLICK))
+		if(using_special)
+			return
+		using_special = TRUE
+		icon_state = "spacedragon_gust"
+		add_dragon_overlay()
+		useGust(0)
+	..()
```

Comment:
Same advice here--I think it's weird when right-click also does the same thing as left-click, that's a side effect of BYOND processing clicks through the same chain.